docs/releases/3.1.txt

```diff
@@ -102,19 +102,14 @@ Minor features
 
 * Select2 library is upgraded from version 4.0.7 to 4.0.13.
 
-:mod:`django.contrib.admindocs`
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-* ...
-
 :mod:`django.contrib.auth`
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 * The default iteration count for the PBKDF2 password hasher is increased from
   180,000 to 216,000.
 
-* Added the :setting:`PASSWORD_RESET_TIMEOUT` setting to define the number of
-  seconds a password reset link is valid for. This is encouraged instead of
+* The new :setting:`PASSWORD_RESET_TIMEOUT` setting allows to define the number
+  of seconds a password reset link is valid for. This is encouraged instead of
```

Comment:
`of deprecated PASSWORD_RESET_TIMEOUT_DAYS` → `of the deprecated PASSWORD_RESET_TIMEOUT_DAYS setting`